R/infrastructure.R

```diff
@@ -33,6 +39,12 @@ use_testthat <- function(pkg = ".") {
     pkg = pkg
   )
 
+  if (use_catch) {
```

Comment:
Yeah, that's sort of what I was thinking - presumably it'd call use_rcpp() too since there's the breaking edge case where catch is called but /src/ doesn't exist? Unless I'm horribly overthinking it.